rclcpp/src/rclcpp/node_options.cpp

```diff
@@ -198,6 +199,19 @@ NodeOptions::use_global_arguments(bool use_global_arguments)
   return *this;
 }
 
+bool
+NodeOptions::enable_rosout() const
+{
+  return this->enable_rosout_;
+}
+
+NodeOptions &
+NodeOptions::enable_rosout(bool enable_rosout)
+{
+  this->enable_rosout_ = enable_rosout;
```

Comment:
@Barry-Xu-2018 thinking about the test made me realize you probably want to `node_options_.reset()` here.